spec.html

```diff
@@ -8113,6 +8113,20 @@
         </emu-alg>
       </emu-clause>
     </emu-clause>
+
+    <emu-clause id="sec-immutable-prototype-exotic-objects">
+      <h1>Immutable Prototype Exotic Objects</h1>
+      <p>An <em>immutable prototype object</em> is an exotic object that disallows mutation of its prototype.</p>
```

Comment:
I think this should be a dfn not an em.

Throughout the document you are inconsistent as to whether it's "immutable prototype exotic object" or "immutable prototype object" or "exotic immutable prototype object".